lib/CppInterOp/Compatibility.h

```diff
@@ -246,16 +263,78 @@ createClangInterpreter(std::vector<const char*>& args) {
   (*ciOrErr)->LoadRequestedPlugins();
   if (CudaEnabled)
     DeviceCI->LoadRequestedPlugins();
+
+#ifdef CPPINTEROP_WITH_OOP_JIT
+  std::unique_ptr<llvm::orc::LLJITBuilder> JB;
+
+  if (outOfProcess) {
+    std::string OOPExecutor =
+        std::string(LLVM_SOURCE_DIR) + "/build/bin/llvm-jitlink-executor";
+    bool UseSharedMemory = false;
+    std::string SlabAllocateSizeString = "";
+    std::unique_ptr<llvm::orc::ExecutorProcessControl> EPC;
+    
+    // Create pipes for child process I/O control
+    int stdout_pipe[2], stderr_pipe[2], stdin_pipe[2];
+    if (pipe(stdout_pipe) != 0 || pipe(stderr_pipe) != 0 || pipe(stdin_pipe)) {
+      llvm::errs() << "Failed to create pipes for child process I/O\n";
+      return nullptr;
+    }
+    
+    EPC = ExitOnError(
+        launchExecutor(OOPExecutor, UseSharedMemory, SlabAllocateSizeString, 
+                      /*stdin_pipe[0]*/0, stdout_pipe[1], stderr_pipe[1]));
+
+    close(stdout_pipe[1]);
+    close(stderr_pipe[1]);
+    // close(stdin_pipe[1]);
+
+    m_child_stdout_fd = stdout_pipe[0];
+    m_child_stderr_fd = stderr_pipe[0];
+    // m_child_stdin_fd = stdin_pipe[1];
+
+#ifdef __APPLE__
+    std::string OrcRuntimePath =
+        std::string(LLVM_SOURCE_DIR) +
+        "/build/lib/clang/20/lib/darwin/liborc_rt_osx.a";
+#else
+    std::string OrcRuntimePath =
+        std::string(LLVM_SOURCE_DIR) +
+        "/build/lib/x86_64-unknown-linux-gnu/liborc_rt.a";
```

Comment:
The path on my Linux machine is:
```suggestion
        "/build/lib/clang/20/lib/linux/liborc_rt-x86_64.a ";
```